samples/drivers/adc/src/main.c

```diff
@@ -146,16 +65,14 @@ void main(void)
 				printk("%d", sample_buffer[0]);
 			}
 
-			/*
-			 * Convert raw reading to millivolts if the reference
-			 * voltage is known.
-			 */
-			if (vref_mv[i] > 0) {
-				print_millivolts(sample_buffer[0],
-						 vref_mv[i],
-						 sequence.resolution,
-						 &adc_channels[i]);
+			/* conversion to mV may not be supported, skip if not */
+			val_mv = sample_buffer[0];
+			err = adc_raw_to_millivolts_dt(&adc_channels[i],
+						       &val_mv);
+			if (err == 0) {
+				printk(" = %d mV", val_mv);
```

Comment:
This is a sample, should be interesting to tell users that mV conversion was skip or it is not available for driver foo?

Reply:
good point, improved